src/java.desktop/share/classes/javax/swing/plaf/basic/BasicSplitPaneDivider.java

```diff
@@ -376,7 +376,7 @@ public void paint(Graphics g) {
 
     /**
      * Messaged when the oneTouchExpandable value of the JSplitPane the
-     * receiver is contained in changes. Will create the
+     * divider is contained in changes. Will create the
```

Comment:
I guess "value of the current JSplitPane divider" makes more sense. 
Also, I guess leftButton,rightButton wll be created not only if they are null but also when oneTouchExpandable property is set as per code..It might seem to user from current wording that left/right button will be set if they are null even if oneTouchExpandable is not set.

Reply:
Messaged when XXXX is little bit confusing but since it is used at many places we can alter them may be as part of another bug.

As @prsadhuk mentioned we can update second statement as "Creates <code>leftButton</code> and <code>rightButton</code> if they are null and corresponding <code>JSplitPane</code> supports oneTouchExpandable buttons"

Its better to use "divider" instead of receiver for same reasons in third statement and update "invalidates" -> "Invalidates".

Since this just clarifies the behavior of this function and nothing is changed from compatibility side. I dont think it needs a CSR.